### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by teammaxxer** [darktrojan/openwith#39]
How about docs, so that people like me know how the arguments works, and what is used for the url
example: %url%

**Comment by darktrojan** [comment]
Yeah, I'll admit documentation is something I haven't really had time for. :-/

`%s` should do what you want.

**Comment by darktrojan** [comment]
Made some progress on the wiki! https://github.com/darktrojan/openwith/wiki
